go-controller/pkg/config/config.go

```diff
@@ -918,6 +918,27 @@ func buildDefaultConfig(cli, file *config) error {
 	return nil
 }
 
+// UseIPv6 returns true if ovn-kubernetes is configured in IPv6 mode, false for IPv4
+func UseIPv6() bool {
+	if len(Default.ClusterSubnets) < 1 {
+		logrus.Warningf("Unable to determine IPv4 vs IPv6 because no cluster subnets configured")
+		return false
+	}
+	if Default.ClusterSubnets[0].CIDR.IP.To4() != nil {
+		return false
+	}
+	return true
+}
```

Comment:
Is there any case where this check won't work?  where the config won't be available?